internal/kafka/internal/services/kafka.go

```diff
@@ -1428,15 +1447,11 @@ func (k *kafkaService) ManagedKafkasRoutesTLSCertificate(kafkaRequest *dbapi.Kaf
 		return err
 	}
 
-	values := map[string]interface{}{
-		"kafkas_routes_base_domain_name":        kafkaRequest.KafkasRoutesBaseDomainName,
-		"kafkas_routes_base_domain_tls_crt_ref": certManagementOutput.TLSCertRef,
-		"kafkas_routes_base_domain_tls_key_ref": certManagementOutput.TLSKeyRef,
-	}
+	kafkaRequest.KafkasRoutesBaseDomainTLSKeyRef = certManagementOutput.TLSKeyRef
+	kafkaRequest.KafkasRoutesBaseDomainTLSCrtRef = certManagementOutput.TLSCertRef
 
-	dbConn := k.connectionFactory.New().Model(kafkaRequest)
-	if err := dbConn.Updates(values).Error; err != nil {
-		return errors.NewWithCause(errors.ErrorGeneral, err, "failed to update kafka %q", kafkaRequest.ID)
+	if svcErr := k.Update(kafkaRequest); svcErr != nil {
+		return svcErr
 	}
 
 	logger.Logger.Infof("management of tls certificate for the domain %q of kafka with id %q successfully finished", kafkaRequest.KafkasRoutesBaseDomainName, kafkaRequest.ID)
```

Comment:
I think this log is not accurate as now it is async and it's not finished by that point

Reply:
good point.